app/models/registration.rb

```diff
@@ -0,0 +1,51 @@
+class Registration
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  SUBMIT_MAPPINGS = HashWithIndifferentAccess.new(
```

Comment:
Mas pravdu, spravme z toho jeden gdoc kam to pojde vsetko. Toto len zbytocne vsetko komplikuje.

Reply:
skor by som asi spravil view, ako uz teraz je admin autoform alebo datahub, a k tomu tabulku.. dat to vsetko do jedneho google sheetu si myslim ze zuzke zas skomplikuje zivot. povedz

Reply:
Myslim, ze nekomplikuje ale spytaj sa jej.

Reply:
Admina urcite na toto nerobime. Nie nie.